tests/test_views.py

```diff
@@ -125,7 +125,7 @@ def test_success_template_change(self):
 
     def test_success_template_change_with_gzip(self):
         # https://github.com/typeddjango/django-stubs/pull/1421
```

Comment:
This comment can also be removed.

Reply:
Thanks for spotting, removed in b07576a8b048830b207a67e2a0222da36d9d2f93